registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/resource/AspspOutboundResource.java

```diff
@@ -0,0 +1,35 @@
+package de.adorsys.registry.manager.resource;
+
+import de.adorsys.registry.manager.client.AspspOutboundClient;
+import de.adorsys.registry.manager.service.AspspOutboundService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+@RestController
+@RequestMapping("/v1/aspsps/outbound")
+public class AspspOutboundResource {
+
+    private AspspOutboundService aspspOutboundService;
+    private AspspOutboundClient aspspOutboundClient;
+
+    public AspspOutboundResource(AspspOutboundService aspspOutboundService, AspspOutboundClient aspspOutboundClient) {
+        this.aspspOutboundService = aspspOutboundService;
+        this.aspspOutboundClient = aspspOutboundClient;
+    }
+
+    @GetMapping("/export")
+    public ResponseEntity<Void> exportData() throws IOException {
+        aspspOutboundClient.exportFile(aspspOutboundService.exportData());
+
+        return ResponseEntity.noContent().build();
+    }
+
+    @GetMapping("/import")
```

Comment:
Should be post method. allow doing that only with the role `DEPLOYER`. the same for the previous one. for details see controllers in the develop branch

Reply:
done